src/layout/Likert/RepeatingGroupsLikertContainerTestUtils.tsx

```diff
@@ -254,28 +254,23 @@ export function ContainerTester(props: { id: string }) {
 }
 
 export const validateTableLayout = (questions: IQuestion[], options: IOption[]) => {
-  screen.getByRole('table');
+  screen.getByRole('group');
 
   for (const option of defaultMockOptions) {
-    const columnHeader = screen.getByRole('columnheader', {
-      name: new RegExp(option.label),
-    });
-    expect(columnHeader).toBeInTheDocument();
+    const allElements = screen.getAllByRole('radio', { name: option.label });
+    for (const element of allElements) {
+      expect(element).toBeInTheDocument();
```

Comment:
Each radio button will place an element with the word "Bra" in the dom. We don't have any role to distinguish this text on any more.